### PR TITLE
Badge updates - new IA and visuals

### PR DESCRIPTION
**Pull request by chri2547**
image

### CODE REVIEW

File: _component_design/badge.md [hunk 1]
```diff
@@ -18,32 +18,106 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/badges/
...
-## Introduction
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use a badge when you want to signal to the user that a new notification, alert, message, chat, comment, or other type of change has taken place. A badge can appear in a navigation menu, tab, or the body of a control panel page.
+
+Use tabs sparingly as overuse can lead to *notification fatigue* where the user fails to react to the badge.
```
Comment (araiford) [inline _component_design/badge.md]: `tabs` => `badges`

**Comment by cyrichardson** [comment]
@chri2547 On it.

File: _component_design/badge.md [hunk 2]
```diff
@@ -81,18 +155,22 @@ Use a 2px border on all sides between the number and the edge of the circle.
...
+
+**With an icon**: Overlap the badge on the icon by 2px and ensure that the icon can still be seen clearly. A notification badge should not contain a **0**. If there are no notifications, do not display a badge.
 
-Notification badges may be used in tabs to bring attention to new items in unselected tabs. Never use a notification badge on the selected tab because any new items in a tab section should have their own styling, such as bolded text, to indicate 'unread' or 'new' statuses.
+- Limit the number of characters in a badge to three, including two numbers and a plus (+) sign. Use a 2px border on all sides between the number and the edge of the circle.
 
-The [tabs component]({{site.baseurl}}/components/tabset.html) states that there should be 24px in between tabs. In the case of a tab with a notification badge, start measuring the 24px after the badge as demonstrated in the "With Spacing Guidelines" image to the right.
+**On a tab**: Add a badge to a tab to bring attention to new items in unselected tabs. Do not use a badge on the selected tab because new items in a tab have their own styling, such as bolded text, to indicate **Unread** or **New** status.
```
Comment (cyrichardson) [inline _component_design/badge.md]: , such as bold text,

File: _component_design/badge.md [hunk 3]
```diff
@@ -18,32 +18,106 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/badges/
...
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best Practices
+
+Consider the following best practices when you construct badges:
+
+- When the user clicks the badge, the user is taken to the section to which the badge is applied.
```
Comment (cyrichardson) [inline _component_design/badge.md]: the user moves to the section
(active voice rather than passive)

File: _component_design/badge.md [hunk 4]
```diff
@@ -18,32 +18,106 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/badges/
...
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### When to use
+### Composition
 
-Use notification badges in navigation menus, tabs, or the page body to signal a new notification, alert, message, chat, comment, or other type of change to an area of the site. Depending on the use case, there may or may not be a number of notifications written within the badge.
+A badge contains the following elements:
 
-Studies of social apps report that notification badges can trigger dopamine releases due to social rewards, but overuse can make the user numb to them. So use badges wisely.
+- **Number (required)**: The number represents the amount of new items available for the user to acknowledge.
```
Comment (cyrichardson) [inline _component_design/badge.md]: The number represents how many new items available

File: _component_design/badge.md [hunk 5]
```diff
@@ -18,32 +18,106 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/badges/
...
+- **Icon (optional)**: Use an icon when you want to visually indicate the type of activity the badge represents. For example, use a bell icon to represent alerts.
```
Comment (cyrichardson) [inline _component_design/badge.md]: Use an icon when you want to indicate visually the type of activity the badge represents. 
(split initiative)

File: _component_design/badge.md [hunk 6]
```diff
@@ -81,18 +155,22 @@ Use a 2px border on all sides between the number and the edge of the circle.
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### In tabs
+## Variations
+
+Considering the following guidelines when using icons and placing a badge on a tab:
+
+**With an icon**: Overlap the badge on the icon by 2px and ensure that the icon can still be seen clearly. A notification badge should not contain a **0**. If there are no notifications, do not display a badge.
```
Comment (cyrichardson) [inline _component_design/badge.md]: that the icon is still seen clearly.
(active voice)

File: _component_design/badge.md (hunk 4 above)
Comment (cyrichardson) [reply _component_design/badge.md]: The number represents how many new items are available

File: _component_design/badge.md (hunk 2 above)
Comment (chri2547) [reply _component_design/badge.md]: Done.

File: _component_design/badge.md (hunk 3 above)
Comment (chri2547) [reply _component_design/badge.md]: Done.

File: _component_design/badge.md (hunk 4 above)
Comment (chri2547) [reply _component_design/badge.md]: Done.

File: _component_design/badge.md (hunk 5 above)
Comment (chri2547) [reply _component_design/badge.md]: Done.

File: _component_design/badge.md (hunk 6 above)
Comment (chri2547) [reply _component_design/badge.md]: Done.

**Pull request merged**